WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -3,30 +3,30 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in CoinJoin">
-    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, that implements trustless CoinJoin over the Tor anonymity network.">
+    <meta name="title" content="Wasabi Wallet - Bitcoin privacy wallet with built-in coinjoin">
+    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for desktop, that implements trustless coinjoin over the Tor anonymity network.">
```

Comment:
I believe Desktop should be capitalized here.

Reply:
Why? 
It looks incorrect to me. Is it?
If the point is to attract attention to the word, spelling it wrong doesn't sound like the way to go.

Reply:
I'd say, lowercase d,: **desktop**